### DISCUSSION TITLE
Login redirect Curso Fullstack

### DISCUSSION THREAD

**Discussion by tiaotj** [balta-io/forum#718] [Tira Dúvidas] [answered]
Não entendi bem o que fiz de errado, mais se eu não tiver logado, não consigo ir pra página de cadastrado porque ele sempre redireciona pra pagina de login.
 
Esse é meu MainLayout
![image](https://github.com/user-attachments/assets/13dce76f-49b0-421b-bcc4-fe46fde28875)

esse é o loginRedirect
![image](https://github.com/user-attachments/assets/b0568d66-2713-4a32-a337-65223bfa5c50)

**Comment by tiaotj** [comment, marked as answer]
Consegui, estava faltando configurar o layout da pagina de registro
@layout HeadlessLayout

**Reply by andrebaltieri** [reply]
Showww!!!
